src/read.c

```diff
@@ -1593,9 +1615,15 @@ static avifResult avifDecoderFindMetadata(avifDecoder * decoder, avifMeta * meta
 
             // Advance past Annex A.2.1's header
             BEGIN_STREAM(exifBoxStream, exifContents.data, exifContents.size, &decoder->diag, "Exif header");
-            uint32_t exifTiffHeaderOffset;
-            AVIF_CHECKERR(avifROStreamReadU32(&exifBoxStream, &exifTiffHeaderOffset),
-                          AVIF_RESULT_BMFF_PARSE_FAILED); // unsigned int(32) exif_tiff_header_offset;
+#if defined(AVIF_ENABLE_EXPERIMENTAL_AVIR)
+            if (!meta->fromConi)
+#endif
+            {
+                // The CondensedImageBox does not signal the exifTiffHeaderOffset.
```

Comment:
Nit: Move this comment before the `if (!meta->fromConi)` line (1619) or at the end of that line.